crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -17,8 +17,10 @@ bloomfilter = { version = "1.0.5", features = [ "serde" ] }
 derive_more = "0.99.11"
 futures = "0.3"
 ghost_actor = "0.3.0-alpha.1"
+governor = "0.3.2"
+itertools = "0.10"
 kitsune_p2p_mdns = { version = "0.0.1", path = "../mdns" }
-kitsune_p2p_types = { version = "0.0.2-dev.0", path = "../types" }
+kitsune_p2p_types = { version = "0.0.2-dev.0", path = "../types", features = ["test_utils"] }
```

Comment:
I don't think we want the `test_utils` feature always on... this should be turned on in via a higher level `test_utils` feature in this crate.